projects/OG-Analytics/src/main/java/com/opengamma/analytics/financial/interestrate/future/provider/InterestRateFutureOptionMarginTransactionNormalSmileMethod.java

```diff
@@ -78,4 +81,73 @@ public SurfaceValue presentValueNormalSensitivity(final InterestRateFutureOption
     return securitySensitivity;
   }
 
+  /**
+   * Computes the present value gamma of a transaction.
+   * This is with respect to futures rate
```

Comment:
The Gamma (second order derivative) is with respect to the futures price (not rate).